packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v2.3.0-deploy-contract-from-json-private.test.ts

```diff
@@ -0,0 +1,278 @@
+
+  const connector1 = await pluginFactoryLedgerConnector.create({
+    instanceId: connectorInstanceId1,
+    pluginRegistry: pluginRegistry1,
+    rpcApiHttpHost: rpcApiHttpHostMember1,
+    privateUrl: keys.quorum.member1.privateUrl,
+    logLevel,
+  });
+  t.ok(connector1, "connector1 truthy OK");
+  test.onFinish(() => connector1.shutdown());
+  pluginRegistry1.add(connector1);
+
+  const connector2 = await pluginFactoryLedgerConnector.create({
+    instanceId: connectorInstanceId2,
+    pluginRegistry: pluginRegistry2,
+    rpcApiHttpHost: rpcApiHttpHostMember2,
+    privateUrl: keys.quorum.member3.privateUrl,
+    logLevel,
+  });
+  t.ok(connector2, "connector2 truthy OK");
+  test.onFinish(() => connector2.shutdown());
+  pluginRegistry2.add(connector2);
+
+  const connector3 = await pluginFactoryLedgerConnector.create({
+    instanceId: connectorInstanceId3,
+    pluginRegistry: pluginRegistry3,
+    rpcApiHttpHost: rpcApiHttpHostMember3,
+    privateUrl: keys.quorum.member3.privateUrl,
+    logLevel,
+  });
+  t.ok(connector3, "connector3 truthy OK");
+  test.onFinish(() => connector3.shutdown());
+  pluginRegistry3.add(connector3);
+
+  await connector1.onPluginInit();
+  await connector2.onPluginInit();
+  await connector3.onPluginInit();
+
+  const accountKeyPath = path.resolve(
+    __dirname,
+    "../../../../resources/network-files/member1",
+    "accountkey",
+  );
+  const accountKey = JSON.parse(fs.readFileSync(accountKeyPath, "utf8"));
+  const signingAccount = webQJsMember1.eth.accounts.decrypt(accountKey, "");
+
+  const accountAddress = keys.quorum.member1.accountAddress;
+  const txCount = await webQJsMember1.eth.getTransactionCount(
+    `0x${accountAddress}`,
+  );
+
+  const deployRes = await connector1.deployContract({
+    bytecode: HelloWorldContractJson.bytecode,
+    contractName: HelloWorldContractJson.contractName,
+    privateTransactionConfig: {
+      privateFrom: keys.tessera.member1.publicKey,
+      privateFor: [
+        keys.tessera.member1.publicKey,
+        keys.tessera.member2.publicKey,
+        keys.tessera.member3.publicKey,
+      ],
+    },
+    web3SigningCredential: {
+      secret: keys.quorum.member1.privateKey,
+      type: Web3SigningCredentialType.PrivateKeyHex,
+      ethAccount: JSON.stringify(signingAccount),
+    },
+    keychainId: keychain1.getKeychainId(),
+    gas: 3000000,
+    gasPrice: 0,
+    nonce: txCount,
+    value: 0,
+  });
+
+  console.log(deployRes);
+
+  const contractDeployReceipt = await webQJsMember1.eth.getTransactionReceipt(
+    deployRes.transactionReceipt.transactionHash,
+  );
+
+  t.ok(contractDeployReceipt, "contractDeployReceipt truthy OK");
+  const receipt = contractDeployReceipt;
+  const { contractAddress } = receipt;
+  t.comment(`Private contract address: ${contractAddress}`);
+  t.ok(contractAddress, "contractAddress truthy OK");
+
+  {
+    const member1Contract = new webQJsMember1.eth.Contract(
+      HelloWorldContractJson.abi as any,
```

Comment:
@travis-payne Can these be casted to something more specific like `AbiItem[]` (just going from memory, might be something else that it wants)

Reply:
Fixed